test/test_helper.rb

```diff
@@ -131,10 +131,10 @@ def create_and_login_additional_users(num)
     end
     visit login_path
 
-    within(".login_form") do
-      fill_in "Username", with: "name#{id}"
-      fill_in "Password", with: "password#{id}"
-      click_button "Login"
+    within('.login_form') do
+      fill_in 'Username', with: "name#{id}"
+      fill_in 'Password', with: "password#{id}"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.